src/db/models/order.ts

```diff
@@ -0,0 +1,45 @@
+"use strict";
+import { Model } from "sequelize";
+import { OrderAttributes } from "../modelsType";
+
+module.exports = (sequelize: any, DataTypes: any) => {
+  class Order extends Model implements OrderAttributes {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    id!: number;
+    uId!: number;
+    pId!: number;
```

Comment:
Please use full id fields, e.g. it is better to have `userId` in the code, but in the DB - `user_id`.